e2e-polybft/cardanofw/test-apex-system.go

```diff
@@ -211,14 +212,56 @@ func (a *ApexSystem) InitContracts(ctx context.Context) error {
 	return nil
 }
 
+func (a *ApexSystem) GetTokenName(chainId string, networkType cardanowallet.CardanoNetworkType) string {
```

Comment:
consider passing `t *testing.T` as parameter (`t.Helper()` is mandatory) and do  require.NoError instead of _